amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java

```diff
@@ -765,38 +760,27 @@ public ResourceService getResourceService() {
     public void setLatestActivityId(Long latestActivityId) {
         this.latestActivityId = latestActivityId;
     }
-    
+
     /**
-     * Get the result of import/update activity in JsonBean format
+     * Get the result of import/update activity
      *
-     * @return JsonBean the result of the import or update action
+     * @return JsonApiResponse the result of the import or update action
      */
-    public JsonBean getResult() {
-        JsonBean result = new JsonBean();
-        if (errors.size() == 0 && newActivity == null) {
-            // no new activity, but also errors are missing -> unknown error
-            result.set(EPConstants.ERROR, ApiError.toError(ApiError.UNKOWN_ERROR).getErrors());
-        } else if (errors.size() > 0) {
-            result.set(EPConstants.ERROR, ApiError.toError(errors.values()).getErrors());
-            result.set(ActivityEPConstants.ACTIVITY, newJson);
-        } else {
-            List<JsonBean> activities = null;
-            if (newActivity != null && newActivity.getAmpActivityId() != null) {
-                // editable, viewable, since was just created/updated
-                activities = Arrays.asList(ProjectList.getActivityInProjectListFormat(newActivity, true, true));
-            }
-            if (activities == null || activities.size() == 0) {
-                result.set(EPConstants.ERROR, ApiError.toError(ApiError.UNKOWN_ERROR).getErrors());
-                result.set(ActivityEPConstants.ACTIVITY, newJson);
-            } else {
-                result = activities.get(0);
```

Comment:
I don't see how JsonApiResponse will display the result when there are no errors.

Reply:
The `details` is computed to activity summary Map, that is configured as `JsonApiResponse.details` and reported unwrapped.

Reply:
ok